src/views/DissolutionFirm/DissolutionFirm.vue

```diff
@@ -313,7 +313,7 @@ export default class DissolutionFirm extends Mixins(DateMixin, EnumMixin) {
 
   /** Is true when the completing party conditions are not met. */
   get isCompletingPartyInvalid ():boolean {
-    return this.getValidateSteps && (!this.getCompletingParty.valid)
+    return this.getValidateSteps && (!this.completingPartyValid) && this.isRoleStaff
```

Comment:
You don't need the internal braces, but I'd like to see outer braces so this looks like a "single return value".